ic-os/defs.bzl

```diff
@@ -183,8 +192,8 @@ def icos_build(
                 for k, v in (
                     image_deps["bootfs"].items() + [
                         (version_txt, "/version.txt:0644"),
-                        (extra_boot_args, "/extra_boot_args:0644"),
-                    ] + ([(boot_args, "/boot_args:0644")] if "boot_args_template" in image_deps else [])
+                    ] + ([(extra_boot_args, "/extra_boot_args:0644")] if "boot_args_template" not in image_deps else []) +
```

Comment:
In looking at https://github.com/dfinity/ic/pull/5437 I realized with this branch we stop passing `extra_boot_args`, and break backwards compatibility with old bootloaders. @andrewbattat will fixup in the next PR.

@frankdavid